aws/README.md

```diff
@@ -38,7 +38,7 @@ There is a `BenchmarkLogger` which must be used when logging any benchmak data.
 Use the following commands to push the image to dockerhub. Avoid building the image using `docker-compose` since it doesn't the ignore files specified in `.dockerignore`.
 ```
 cd /path/to/HoneyBadgerMPC
-docker build -t honeybadger .
+docker build -t honeybadger . --build-arg BUILD=dev
```

Comment:
what's that for?

Reply:
The Dockerfile has an argument to pip install which works differently for tests, docs and dev. This needs to be passed to build the correct image.